### DISCUSSION TITLE
How can I change the default error messages?

### DISCUSSION THREAD

**Discussion by CaioCamatta** [django-recaptcha/django-recaptcha#251] [Q&A] [answered]
I tried the following, but it doesn't seem to work.

```
captcha_default_errors = {
    'required': 'some message here',
    'invalid': 'some message here'
}
```
 `captcha = ReCaptchaField(error_messages=captcha_default_errors)`

**Comment by pringelmann** [comment]
Hi Caio,

I will investigate this issue over the next day or two and get back to you.

**Comment by pringelmann** [comment]
@CaioCamatta 

I cannot seem to replicate this issue.
Can you please provide me with some more detailed information? (including your project settings please)

**Comment by CaioCamatta** [comment]
@Pringels 

How should I change the default error messages? Is that the right way? 
Maybe I'm just doing it the wrong way.

**Comment by pringelmann** [comment]
@CaioCamatta 

That is the correct way to do it. Can you provide me with a bit more information on the rest of your form code?

**Comment by manikos** [comment, marked as answer]
Here's a peak at my code which overrides the default error messages:

```
# forms.py

from captcha.fields import ReCaptchaField
from django.utils.translation import ugettext_lazy as _

captcha = ReCaptchaField(attrs={'size': 'normal', 'callback': 'captchaCallback'}, 
error_messages={'required': _("You're not a robot, right?")})
```

```
# submit_form.html

...
{% if my_form.captcha.errors %}
    <div class="my_form_error my-captcha-error">{{ my_form.captcha.errors }}</div>
{% endif %}
<span>{{ my_form.captcha }}</span>
```

It works great on me.
